csrankings-g.csv

```diff
@@ -844,7 +844,7 @@ Guillermo Alberto Pérez,University of Antwerp,https://www.uantwerpen.be/en/staf
 Guillermo Gallego 0002,TU Berlin,https://www.eecs.tu-berlin.de/menue/einrichtungen/professuren/professorinnen/gallego/parameter/en/,v0_XxF0AAAAJ
 Guillermo Gallego Bonet,TU Berlin,https://www.eecs.tu-berlin.de/menue/einrichtungen/professuren/professorinnen/gallego/parameter/en/,v0_XxF0AAAAJ
 Guillermo Payá Vayá,TU Braunschweig,https://www.tu-braunschweig.de/eis/paya-vaya,uJLGjFUAAAAJ
-Guillermo Sapiro,Duke University,https://ece.duke.edu/faculty/guillermo-sapiro,ISRNX3gAAAAJ
+Guillermo Sapiro,Princeton University,https://ece.princeton.edu/people/gsapiro,ISRNX3gAAAAJ

```

Comment:
@emeryberger Before merging this, please double check whether in industry: Distinguished Engineer, Health AI Lead, Apple

Reply:
Article seems to imply full-time employment at Princeton: https://engineering.princeton.edu/news/2024/10/22/nasa-fda-sapiro-delivers-high-impact-technologies